src/react/src/__tests__/components/FacilityDetailsClaimedInfo.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import FacilityDetailsClaimedInfo from '../../components/FacilityDetailsClaimedInfo';
+import renderWithProviders from '../../util/testUtils/renderWithProviders';
+
+describe('FacilityDetailsClaimedInfo component', () => {
+    test('test facility details presence', () => {
+      const data = {
```

Comment:
Note, that we use here the `data` object as it was fetched by the request and passed to the component. FacilityDetailsClaimedInfo doesn't interact with Redux state.

Reply:
Thanks